openapi/connector_mgmt.yaml

```diff
@@ -957,17 +1299,16 @@ components:
       required:
         - name
```

Comment:
As a side note, we require `name` to be provided, but we do not validate for uniqueness

Reply:
there is one already

Reply:
Actually, I did create a constraint in the DB, so user will see an error, although about a DB constraint violation. :)